### ISSUE TITLE
Exceptions, thrown in templates not showing

### ISSUE THREAD

**Issue by thexpand** [zendframework/zend-mvc#157]
Since release-2.7.8 there is a problem when an error occurs in the template (even a syntax or parse error). The framework says that an error occurred and "No exception is available".
Sometimes it is frustrating, because I can't even see on which line is the error. I am using the latest PHP7 version (v7.0.7) on Ubuntu 16.10.

**Comment by zf2timo** [comment]
Maybe this is related to #146

**Comment by kokspflanze** [comment]
hi,

i reproduce the problem, form my site it looks like that in your error template is a check like

``` php
<?php if(isset($this->exception) && $this->exception instanceof Exception): ?>
```

the problem is that the $this->exception is from class `Error`.

so change it to smth like that

``` php
<?php if(isset($this->exception) && ($this->exception instanceof Exception || $this->exception instanceof Error)): ?>
```

hope i can help you.

about https://github.com/zendframework/zend-mvc/issues/146 this result in http error code 500 with a error in your webserver logs.

Tested on PHP 7.0.7 CentOS 7

**Comment by thexpand** [comment]
I don't have any checks in my templates. I'm just calling a method on a non-object. I don't get an error - instead, I get the message: "No exception available" and I have to debug every single line.

**Comment by weierophinney** [comment]
@thexpand It looks like it's solved in #153; I'll be merging later today and issuing a release.

**Comment by thexpand** [comment]
It's working like a charm! Thanks.
